app/forms/respondent_form.rb

```diff
@@ -95,4 +93,8 @@ def reset_work_address!
   def reload_addresses
     target.addresses.reload
   end
+
+  def validate_acas_reason_presence
```

Comment:
is this used ?

Reply:
no, should have removed that. removed now.